### PR TITLE
cli: deploy aws csi driver per default

### PR DESCRIPTION
**Pull request by msanft**
### Context

The AWS CSI driver was not deployed per default so far, but should be.

### Proposed change(s)

- Add the AWS CSI driver Helm chart to our vendored charts.
- Add the config key `aws.deployCSIDriver` to the Constellation config.

### Checklist

- [x] Update [docs](https://github.com/edgelesssys/constellation/tree/main/docs) - #1974 
- [x] Add labels (e.g., for changelog category)
- [x] Is PR title adequate for changelog?
- [x] Link to Milestone

### CODE REVIEW

File: cli/internal/helm/charts/edgeless/constellation-services/charts/aws-csi-driver/templates/tests/helm-tester.yaml
Comment (daniel-weisse) [inline cli/internal/helm/charts/edgeless/constellation-services/charts/aws-csi-driver/templates/tests/helm-tester.yaml:1]: What is this helm-tester used for?
Seems to run some sort of kubernetes e2e test?

File: cli/internal/helm/update-csi-charts.sh [hunk 1]
```diff
@@ -58,10 +58,14 @@ download_chart() {
   return
 }
 
-## GCP CSI Driver
-download_chart "https://github.com/edgelesssys/constellation-gcp-compute-persistent-disk-csi-driver" "v1.2.0" "charts" "gcp-compute-persistent-disk-csi-driver"
+## AWS CSI Driver
+# TODO(msanft): Update to v1.1.0 once released
+download_chart "https://github.com/edgelesssys/constellation-aws-ebs-csi-driver" "feat/helm/default-storage-classes" "charts/aws-ebs-csi-driver" "aws-csi-driver"
```
Comment (daniel-weisse) [inline cli/internal/helm/update-csi-charts.sh:63]: Fix todo before merging

**Review by daniel-weisse — COMMENTED** [review]
Should run the e2e tests for AWS on this branch to make sure everything is deployed correctly

**Comment by msanft** [comment]
[e2e test](https://github.com/edgelesssys/constellation/actions/runs/5410262468)

**Review by daniel-weisse — APPROVED** [review]

**Pull request merged**